lang/java.md

```diff
@@ -0,0 +1,114 @@
+---
+    lang: Java
+    syntax: java
+    title: Java is a statically typed JVM based programming language.
+    author: Khafia Ayyub
```

Comment:
⛔ This should be your GitHub username, not your real name. Otherwise, we don't be able to credit you.

Reply:
@vsnthdev: Shall we take this in a format of:
```
[amino19](https://github.com/amino19)
```
_Just a sweet interference... Don't mind!_